graylog2-web-interface/src/components/common/SourceCodeEditor.tsx

```diff
@@ -297,11 +332,14 @@ class SourceCodeEditor extends React.Component {
                                $resizable={resizable}>
             <AceEditor ref={(c) => {
               this.reactAce = c;
+              // @ts-ignore
               if (innerRef) { innerRef.current = c; }
             }}
                        annotations={annotations}
+                       // @ts-expect-error
```

Comment:
It looks like we can avoid `@ts-expect-error` here by changing `$blockScrolling: 'Infinity'` to `$blockScrolling: Infinity`